action.yaml

```diff
@@ -107,16 +107,23 @@ inputs:
     description: 'skip calling the setup-trivy action to install trivy'
     required: false
     default: 'false'
+  token-setup-trivy:
+    description: >
+      `token-setup-trivy` is required when `github.token` in invalid for `https://github.com` server.
+      See https://github.com/aquasecurity/setup-trivy/?tab=readme-ov-file#install-trivy-with-non-default-token for more details.
+    required: false
+    default: ''
 
 runs:
   using: 'composite'
   steps:
     - name: Install Trivy
       if: ${{ inputs.skip-setup-trivy == 'false' }}
-      uses: aquasecurity/setup-trivy@v0.2.1
+      uses: DmitriyLewen/setup-trivy@2df9f605c29dbee29f7c66a8151e3b97f0874f5d
```

Comment:
Just a note that we should bump this to the next release when the other PR gets merged.

Reply:
Right. I used my fork to test the changes.
After `setup-trivy` is released - I will update this line.

Reply:
Updated in -https://github.com/aquasecurity/trivy-action/pull/421/commits/9960a61d3b34b079b1b0bd246268917e5a7dd6af